src/main/java/org/apache/sling/feature/cpconverter/handlers/slinginitialcontent/SlingInitialContentBundleEntryMetaDataCollector.java

```diff
@@ -190,6 +249,21 @@ private SlingInitialContentBundleEntryMetaData createSlingInitialContentBundleEn
         String repositoryPath = (target != null ? target : "/") + URLDecoder.decode(entryName.substring(pathEntryValue.getPath().length()), "UTF-8");
         return new SlingInitialContentBundleEntryMetaData(targetFile, pathEntryValue, repositoryPath);
     }
+    
+
+    private PathEntry getPathEntryFromRepositoryPath(@NotNull String repositoryPath) {
+
+        if(StringUtils.countMatches(repositoryPath, '/') < 2){
```

Comment:
I find this opaque - please document the reason. Since we use it in two places, it would be good I think to extract to a method or at least have a contant whose name explains what it does.

Reply:
and add a space after if and before { (or let the IDE fix the formatting)